src/tests/Directory.Merged.props

```diff
@@ -0,0 +1,8 @@
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <InMergedTestDirectory>true</InMergedTestDirectory>
+    <BuildAsStandalone Condition="'$(BuildAsStandalone)' == ''">false</BuildAsStandalone>
+
+    <AssemblyName Condition="'$(BuildAsStandalone)' != 'true'">$(MSBuildProjectName.Replace('_il_d', '').Replace('_il_r', ''))</AssemblyName>
```

Comment:
IMO, it would be helpful to have some comments. Especially, to describe why we need this weird `_il_d`/`_il_r` handling.